_rules/role-attribute-valid-value-674b10.md

```diff
@@ -106,20 +106,26 @@ This `role` attribute contains two tokens, and one of these tokens (`img`) is a
 This `role` attribute contains one token, but this token is not a valid role in any of the [WAI-ARIA specifications][].
 
 ```html
-<img role="lnik" src="/test-assets/shared/w3c-logo.png" alt="W3C logo" onclick="location.href='https://www.w3.org/'" />
+<label>Search: <input type="text" role="searchfield" placeholder="Enter 3 or more characters"/></label>
```

Comment:
I think this is an example of where that third assumption might be true. Without the role attribute, this would be a perfectly valid form field.

Reply:
I would argue that the correct (and definitely the intended) role is here `searchbox`, so the third assumption still holds. (and this is indicated by the "Search" label).